cfgov/unprocessed/js/modules/nav-primary.js

```diff
@@ -5,85 +5,98 @@
 
 'use strict';
 
-var $ = require( 'jquery' );
-var es = require( './util/expanded-state.js' );
+var expandedState = require( './util/expanded-state' );
+var domTraverse = require( './util/dom-traverse' );
+
+var _primaryNav;
+var _primaryTrigger;
+var _primaryLinks;
+var _subNavs;
 
 /**
  * Set up DOM references and event handlers.
  */
 function init() {
-  var $primaryNav = $( '.js-primary-nav' );
-  var $primaryTrigger = $( '.js-primary-nav_trigger' );
-  var $primaryLink = $( '.js-primary-nav_link' );
-  var $subNavs = $( '.js-sub-nav' );
-  var $subBack = $( '.js-sub-nav_back' );
-
-  $primaryTrigger.on( 'click', function() {
-    if ( es.get.isThisExpanded( $primaryNav ) ) {
-      es.set.toggleExpandedState( $primaryNav, 'false', function() {
-        $primaryTrigger.focus();
-      } );
-    } else {
-      es.set.toggleExpandedState( $primaryNav, 'true', function() {
-        $primaryLink.first().focus();
-      } );
-    }
-
-    es.set.toggleExpandedState( $primaryTrigger );
-    es.set.toggleExpandedState( $( 'body' ) );
-  } );
+  _primaryNav = document.querySelector( '.js-primary-nav' );
+  _primaryTrigger = document.querySelector( '.js-primary-nav_trigger' );
+  _primaryLinks = _primaryNav.querySelectorAll( '.js-primary-nav_link' );
+  _subNavs = _primaryNav.querySelector( '.js-sub-nav' );
 
-  $primaryLink.on( 'click', function( event ) {
-    event.preventDefault();
-
-    var $this = $( this );
-    var $thisSubNav = $this.siblings( '.js-sub-nav' );
-    var $otherSubNavs = $subNavs.not( $thisSubNav );
-    var firstLink = $thisSubNav.find( 'a' ).first();
-
-    if ( es.get.isOneExpanded( $otherSubNavs ) ) {
-      es.set.toggleExpandedState( $primaryLink, 'false' );
-      es.set.toggleExpandedState(
-        $otherSubNavs,
-        'false',
-        function() {
-          es.set.toggleExpandedState( $thisSubNav );
-          firstLink.focus();
-        }
-      );
-    } else if ( es.get.isThisExpanded( $thisSubNav ) ) {
-      es.set.toggleExpandedState( $thisSubNav, 'false', function() {
-        $this.focus();
-      } );
-    } else {
-      es.set.toggleExpandedState( $thisSubNav, null, function() {
-        firstLink.focus();
-      } );
-    }
+  _primaryTrigger.addEventListener( 'click', _primaryTriggerClicked );
 
-    es.set.toggleExpandedState( $this );
-  } );
+  for ( var i = 0, len = _primaryLinks.length; i < len; i++ ) {
+    _primaryLinks[i].addEventListener( 'click', _primaryLinkClicked );
```

Comment:
How's this look https://github.com/cfpb/cfgov-refresh/commit/375c98ffd9b0a4112018d5ffa8b37ac1563f4268